app/boards/nucleo_h745zi_q.overlay

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2024 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/gpio/gpio.h>
+
+/ {
+	chosen {
+		zephyr,canbus = &fdcan1;
+	};
+
+	cannectivity: cannectivity {
+		compatible = "cannectivity";
+		timestamp-counter = <&counter2>;
+
+		channel0 {
+			compatible = "cannectivity-channel";
+			can-controller = <&fdcan1>;
+			state-gpios = <&gpiob 0 GPIO_ACTIVE_HIGH>;
+			activity-gpios = <&gpioe 1 GPIO_ACTIVE_HIGH>;
+		};
+	};
+};
+
+&zephyr_udc0 {
+	gs_usb0: gs_usb0 {
+		compatible = "gs_usb";
+	};
+};
+
+&fdcan1 {
+	/* Allocate half of the M_CAN message RAM to FDCAN1 */
+	reg = <0x4000a000 0x400>, <0x4000ac00 0x1400>;
+	reg-names = "m_can", "message_ram";
+	bosch,mram-cfg = <0x0 1 1 10 10 0 10 10>;
+	pinctrl-0 = <&fdcan1_rx_pb8 &fdcan1_tx_pb9>;
```

Comment:
How come this is changed from the board defaults of `pinctrl-0 = <&fdcan1_rx_pd0 &fdcan1_tx_pd1>;`?

Reply:
There is only one shield, which patches perfectly the connector on nucleo_h745 board: https://www.waveshare.com/rs485-can-shield.htm and it needs this pinout.
The pd0 and pd1 pinse are default pins nucleo allocates for CAN accroding to their dicumentation. 
Therefore the change is needed.

Reply:
But just because you need to use this particular board with that particular shield does not make it a generic configuration suitable for other users of CANnectivity. Your change makes the configuration _incompatible_ with what others expect from the board schematic.

I suggest keeping this board + shield configuration in your own repository.

Reply:
> But just because you need to use this particular board with that particular shield does not make it a generic configuration suitable for other users of CANnectivity. Your change makes the configuration _incompatible_ with what others expect from the board schematic.
> 
> I suggest keeping this board + shield configuration in your own repository.

It terms of CAN-L and CAN-H pins I may agree, but what about LEDs? see conversation: https://github.com/CANnectivity/cannectivity/pull/31#discussion_r1873082161